targets/psa_switch/psa_switch.cpp

```diff
@@ -206,7 +208,9 @@ PsaSwitch::set_egress_queue_depth(size_t port, const size_t depth_pkts) {
 
 int
 PsaSwitch::set_all_egress_queue_depths(const size_t depth_pkts) {
-  egress_buffers.set_capacity_for_all(depth_pkts);
+  for (uint32_t i = 0; i < max_port; i++) {
+    set_egress_queue_depth(i, depth_pkts);
+  }
```

Comment:
Is this present in simple_switch code as well?  I would guess that when Antonin recently made the changes to allow a configurable max_port value, he might also have allowed the set of valid ports to not always be consecutive from [0, max_port-1], i.e. there might be "holes" in that number range that do not represent valid ports.  Not a big deal, but worth checking.

Reply:
Accidental overwrites occurred, reverted.